test/js/node/fs/fs.test.ts

```diff
@@ -2729,3 +2729,30 @@ it("fs.ReadStream allows functions", () => {
   // @ts-expect-error
   expect(() => new fs.ReadStream(".", {})).not.toThrow();
 });
+
+describe.if(isWindows)("windows path handling", () => {
```

Comment:
Does the Bun test runner treat this as a skipped message on platforms not windows (just to know that the test is there but not being run)?

Reply:
yes